### PR TITLE
changes for repairers lien table rows

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#9105

*Description of changes:*
- table row changes for repairers lien

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/useRegistration.ts [hunk 1]
```diff
@@ -108,6 +108,18 @@ export const useRegistration = () => {
     return false
   }
 
+  const hasRenewal = (
```
Comment (doug-lovett) [inline ppr-ui/src/composables/useRegistration.ts:111]: Is this checking if there has been a previous renewal registration?

**Review by doug-lovett — APPROVED** [review]
Looks good, just one question.

File: ppr-ui/src/composables/useRegistration.ts (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/src/composables/useRegistration.ts:111]: yep. it goes through the children to see if there has been a previous renewal

Comment (doug-lovett) [reply ppr-ui/src/composables/useRegistration.ts:111]: So should it check that registrationClass === 'RENEWAL' or registrationType === 'RE'?

Comment (jordiwes) [reply ppr-ui/src/composables/useRegistration.ts:111]: yes, good point. I've made the fix

**Pull request merged**